DOWNLOAD_STATS.md

```diff
@@ -5,14 +5,30 @@
+| `@nolyfill/array.prototype.join` | [![npm](https://img.shields.io/npm/dt/@nolyfill/array.prototype.join.svg?style=flat-square&logo=npm&logoColor=white&label=total%20downloads&color=333)](https://www.npmjs.com/package/@nolyfill/array.prototype.join) |
+| `@nolyfill/array.prototype.keys` | [![npm](https://img.shields.io/npm/dt/@nolyfill/array.prototype.keys.svg?style=flat-square&logo=npm&logoColor=white&label=total%20downloads&color=333)](https://www.npmjs.com/package/@nolyfill/array.prototype.keys) |
+| `@nolyfill/array.prototype.push` | [![npm](https://img.shields.io/npm/dt/@nolyfill/array.prototype.push.svg?style=flat-square&logo=npm&logoColor=white&label=total%20downloads&color=333)](https://www.npmjs.com/package/@nolyfill/array.prototype.push) |
 | `@nolyfill/array.prototype.reduce` | [![npm](https://img.shields.io/npm/dt/@nolyfill/array.prototype.reduce.svg?style=flat-square&logo=npm&logoColor=white&label=total%20downloads&color=333)](https://www.npmjs.com/package/@nolyfill/array.prototype.reduce) |
+| `@nolyfill/array.prototype.reduceright` | [![npm](https://img.shields.io/npm/dt/@nolyfill/array.prototype.reduceright.svg?style=flat-square&logo=npm&logoColor=white&label=total%20downloads&color=333)](https://www.npmjs.com/package/@nolyfill/array.prototype.reduceright) |
+| `@nolyfill/array.prototype.slice` | [![npm](https://img.shields.io/npm/dt/@nolyfill/array.prototype.slice.svg?style=flat-square&logo=npm&logoColor=white&label=total%20downloads&color=333)](https://www.npmjs.com/package/@nolyfill/array.prototype.slice) |
+| `@nolyfill/array.prototype.some` | [![npm](https://img.shields.io/npm/dt/@nolyfill/array.prototype.some.svg?style=flat-square&logo=npm&logoColor=white&label=total%20downloads&color=333)](https://www.npmjs.com/package/@nolyfill/array.prototype.some) |
+| `@nolyfill/array.prototype.splice` | [![npm](https://img.shields.io/npm/dt/@nolyfill/array.prototype.splice.svg?style=flat-square&logo=npm&logoColor=white&label=total%20downloads&color=333)](https://www.npmjs.com/package/@nolyfill/array.prototype.splice) |
 | `@nolyfill/array.prototype.tosorted` | [![npm](https://img.shields.io/npm/dt/@nolyfill/array.prototype.tosorted.svg?style=flat-square&logo=npm&logoColor=white&label=total%20downloads&color=333)](https://www.npmjs.com/package/@nolyfill/array.prototype.tosorted) |
+| `@nolyfill/array.prototype.unshift` | [![npm](https://img.shields.io/npm/dt/@nolyfill/array.prototype.unshift.svg?style=flat-square&logo=npm&logoColor=white&label=total%20downloads&color=333)](https://www.npmjs.com/package/@nolyfill/array.prototype.unshift) |
+| `@nolyfill/array.prototype.values` | [![npm](https://img.shields.io/npm/dt/@nolyfill/array.prototype.values.svg?style=flat-square&logo=npm&logoColor=white&label=total%20downloads&color=333)](https://www.npmjs.com/package/@nolyfill/array.prototype.values) |
```

Comment:
Nobody uses `array.prototype.values` on npm. It has 0 dependent and its weekly downloads are less than 10.